ddtrace/llmobs/_llmobs.py

```diff
@@ -176,6 +171,13 @@ def enable(
         atexit.register(cls.disable)
         log.debug("%s enabled", cls.__name__)
 
+    @classmethod
+    def _integration_is_enabled(cls, integration):
+        if integration not in SUPPORTED_INTEGRATIONS:
+            log.warning("%s is not a supported LLMObs integration", integration)
```

Comment:
I'm worried this might lead to a lot of warning logs in a user's application, if they use Cohere/Anthropic. Since we're the only ones using this function internally, let's remove the log